src/backends/pgfplots.jl

```diff
@@ -87,17 +87,17 @@ const _pgfplots_linestyles = KW(
 )
 
 const _pgfplots_markers = KW(
-    :none => "mark = none,",
-    :cross => "mark = +,",
-    :xcross => "mark = x,",
-    :utriangle => "mark = triangle*,",
-    :dtriangle => "mark = triangle*,",
-    :ellipse => "mark = o*,",
-    :rect => "mark = square*,",
-    :star5 => "mark = star,",
-    :star6 => "mark = asterisk,",
-    :diamond => "mark = diamond*,",
-    :pentagon => "mark = pentagon*,"
+    :none => "mark = none",
+    :cross => "mark = +",
+    :xcross => "mark = x",
+    :utriangle => "mark = triangle*",
+    :dtriangle => "mark = triangle*",
+    :ellipse => "mark = o*",
+    :rect => "mark = square*",
+    :star5 => "mark = star",
+    :star6 => "mark = asterisk",
+    :diamond => "mark = diamond*",
+    :pentagon => "mark = pentagon*"
```

Comment:
I'll just make some notes of what I might do if I was cleaning up (you don't have to listen to me though).

Here it seems wasteful to include "mark = " in every string when you could do that once outside of the KW.